### DISCUSSION TITLE
Syntax of `CREATE TYPE` vs `CREATE PROPERTY`

### DISCUSSION THREAD

**Discussion by gramian** [ArcadeData/arcadedb#893] [General]
I was wondering about the syntax of the `CREATE TYPE` SQL commands. For example, a document type is created by:
```
CREATE DOCUMENT TYPE doc;
```
whereas a property is created by:
```
CREATE PROPERTY doc.prop STRING;
```
In comparison, the type of a type is placed between `CREATE` and `TYPE` while the type of a property follows the property name. Is there a reason
```
CREATE TYPE doc DOCUMENT;
```
or alternatively
```
CREATE STRING PROPERTY doc.prop;
```
was not selected, so types and properties are created by "similar" commands?

**Comment by lvca** [comment]
Our current syntax was inherited from OrientDB for properties, but I like the one you propose. It could be confusing having both commands (the current one must be left for backward compatibility with ArcadeDB and OrientDB)
